src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
```

Comment:
-Lottery doesnt need to extend a ColorSupplier, it complicates the code and doesn't have any logic behind it
-Lottery class should have an instanse of ColorSupplier class to supply it random colours using a non static method